tests/Integration/Auth/CanFindDirectiveDBTest.php

```diff
@@ -218,7 +256,8 @@ public function testFailsToFindSpecificModelWithFindOrFailFalse(): void
 
         $this->mockResolver(null);
 
-        $this->schema = /** @lang GraphQL */ '
+        $this->schema /** @lang GraphQL */
```

Comment:
It's better not to change coding style in unrelated places.

Reply:
Please revert that part.